serialization.md

```diff
@@ -472,4 +472,41 @@ The resulting Tar archive for `f60c56784b83` has the following entries:
 /etc/.wh.my-app-config
 ```
 
+In addition to expressing that a single entry should be removed from a lower layer, layers may remove all of the children using an opaque whiteout entry.
+An opaque whiteout entry is a file with the name `.wh..wh..opq` indicating that all siblings are hidden in the lower layer.
+Let's take the following base layer as an example:
+
+```
+etc/
+	my-app-config
+bin/
+	my-app-binary
+	my-app-tools
+	tools/
+		my-app-tool-one
+```
+
+If we remove all of the children of `bin/` in the next layer, we would have the following:
+
+```
+bin/
+	.wh..wh..opq
+```
+
+We'll call this the _opaque whiteout_ format.
+Note that this opaque file will apply to _all_ children, including sub-directories, other resources and all descendents.
+Using _explicit whiteout_ files, this would be equivalent to the following:
+
+```
+bin/
+	.wh.my-app-binary
+	.wh.my-app-tools
+	.wh.tools
+```
+
+In this case, a unique whiteout file is generated for each entry.
```

Comment:
this can be in misleading with the example above https://github.com/opencontainers/image-spec/pull/171/files#diff-9f7e4d8b174e71a2ac5d4d28c9548f5cR497